install-dependencies.sh

```diff
@@ -131,6 +131,17 @@ centos8_packages=(
     gcc-toolset-11-libatomic-devel
 )
 
+centos9_packages=(
+    "${redhat_packages[@]}"
+    ninja-build
+    ragel
+    gcc-toolset-13-gcc
+    gcc-toolset-13-gcc-c++
```

Comment:
i think `gcc-toolset-13-gcc-c++` depends on [gcc-toolset-13-gcc](https://pkgs.org/download/gcc-toolset-13-gcc). and as a C++ framework, Seastar should be built with C++. any reason to install gcc explicitly?

Reply:
The same is true with [gcc-toolset-11-gcc-c++](https://almalinux.pkgs.org/8/almalinux-appstream-aarch64/gcc-toolset-11-gcc-c++-11.2.1-9.1.el8.aarch64.rpm.html). `centos8_packages` should be edited as well then, right?

Reply:
we can include that change in another PR.

Hi @JunHe77 i just realized that you added `gcc-toolset-9-gcc` back in https://github.com/scylladb/seastar/commit/627775dfa94bb4fa8e277437239b33a06dedf946#diff-4852d2a5a482159af418383e38099c1321b69c6272981405ed32d629a19de73dR123 . do you happen to remember why that package was added?